diff --git a/src/client/index.js b/src/client/index.js
index 585984d..5463f3f 100644
--- a/src/client/index.js
+++ b/src/client/index.js
@@ -5,12 +5,8 @@ import "./styles/results.scss";
 import "./styles/reset.scss";
 import "./styles/savedTravelResults.scss";
 import img from "./media/placeholder.jpg";
-import { handleSubmit } from "./js/app";
-import { closeButtonEvent } from "./js/app";
-import { saveButtonEvent } from "./js/app";
-import { postData } from "./js/sub-app";
-import { getData } from "./js/sub-app";
-import { updateUI } from "./js/sub-app";
+import { handleSubmit, closeButtonEvent, saveButtonEvent } from "./js/app";
+import { postData, getData, updateUI } from "./js/sub-app";
 
 document.getElementById("submit").addEventListener("click", handleSubmit);
 document.getElementById("save").addEventListener("click", saveButtonEvent);
diff --git a/src/client/styles/style.scss b/src/client/styles/style.scss
index f33df2a..080f41a 100644
--- a/src/client/styles/style.scss
+++ b/src/client/styles/style.scss
@@ -7,6 +7,7 @@ body {
   font-family: -apple-system, BlinkMacSystemFont, sans-serif;
   background: url("../media/placeholder.jpg");
   background-position: center;
-  background-repeat: repeat;
+  background-repeat: no-repeat;
   background-size: cover;
+  background-attachment: fixed;
 }
diff --git a/src/server/server.js b/src/server/server.js
index 8d6a3d9..1281d85 100644
--- a/src/server/server.js
+++ b/src/server/server.js
@@ -26,7 +26,6 @@ app.use(bodyParser.json());
 
 // Cors for cross origin allowance
 const cors = require("cors");
-const { response } = require("express");
 app.use(cors());
 
 // Initialize the main project folder
@@ -51,16 +50,9 @@ app.get("/", function (req, res) {
 // Callback function to complete GET '/all'
 app.get("/all", (req, res) => {
   res.send(projectData);
-  projectData;
 });
 
 app.post("/createTrip", (req, res) => {
-  req.body;
-
-  // targeting the days of date to get the difference
-  req.body.startDate;
-  req.body.endDate;
-  req.body.duration;
 
   const startDays = req.body.startDate.slice(0, 10);
   const endDays = req.body.endDate.slice(0, 10);
@@ -86,7 +78,7 @@ app.get("/geonames", (req, res) => {
       projectData.lat = response.geonames[0].lat;
       projectData.long = response.geonames[0].lng;
 
-      `ProjectData is, ${projectData}`;
+      console.log(`ProjectData is, ${JSON.stringify(projectData,null,2)}`);
       res.send(true);
     })
     .catch((error) => {
@@ -102,7 +94,7 @@ app.get("/weatherBit", (req, res) => {
     const weatherData = response.data;
 
     weatherData.forEach((data) => {
-      if (data.valid_date == projectData.startDate) {
+      if (data.valid_date === projectData.startDate) {
         projectData.description = data.weather.description;
         projectData.temp = data.temp;
         res.send(true);
@@ -125,20 +117,21 @@ app.get("/all", (req, res) => {
   console.log(projectData);
 });
 
-// Test to test the server is workinga
+// Test to test the server is working
 app.get("/test", (req, res) => {
   res.send("Hi, the server is working...");
 });
 
 // Global functions
 const getData = async (url) => {
-  const response = await fetch(url);
   try {
+    const response = await fetch(url);
     const data = await response.json();
     // console.log(data);
     return data;
   } catch (error) {
     console.log("error", error);
+    return error;
   }
 };
 
